static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -143,7 +143,8 @@ define([
                     && !config.page.isAdvertisementFeature
                     && config.page.pageId !== 'offline-page'
                     && !config.page.shouldHideAdverts
-                    && config.page.section !== 'childrens-books-site') {
+                    && config.page.section !== 'childrens-books-site'
+                    && !config.tests.abNewHeaderVariant) {
```

Comment:
This is a discussion for a different time, but do we know how the design on desktop would look with the sticky ad banner?